Base/QTCore/Testing/Cxx/qSlicerUtilsTest1.cxx

```diff
@@ -25,6 +25,7 @@
 #include <QTime>
 
 // CTK includes
+#include <ctkCoreTestingMacros.h>
```

Comment:
Nitpick: Move refactoring to use the macro into its own commit, and only keep addition of `replaceWikiUrlVersion` test to this commit